src/main/java/org/jenkinsci/plugins/workflow/cps/CpsWhitelist.java

```diff
@@ -103,6 +104,9 @@ public boolean permitsStaticMethod(Method method, Object[] args) {
             String driverTo = "org.codehaus.groovy.runtime.";
             if (cn.startsWith(driverFrom)) {
                 try {
+                    if (Modifier.isPrivate(method.getModifiers())) {
+                        return true;
```

Comment:
Probably OK but I want to think a bit about more why this is necessary. `dgm-builder` is not intentionally generating code that uses `SandboxCpsTransformer` hooks so how are these calls going through the sandbox interceptor to begin with?